pkg/asset/ignition/machine/node.go

```diff
@@ -30,8 +31,10 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 		} else {
 			ignitionHost = fmt.Sprintf("api-int.%s:22623", installConfig.ClusterDomain())
 		}
-	case ovirt.Name:
+	case ovirttypes.Name:
 		ignitionHost = fmt.Sprintf("%s:22623", installConfig.Ovirt.APIVIP)
+	case vspheretypes.Name:
```

Comment:
@mtnbikenc @vrutkovs right here is where ignition is broken.  Need to add a check for empty string APIVIP

Reply:
https://github.com/openshift/installer/pull/2956